consensus/core/src/authority_node.rs

```diff
@@ -42,6 +43,21 @@ pub enum ConsensusAuthority {
     WithTonic(AuthorityNode<TonicManager>),
 }
 
+pub struct ConsensusOutput {
+    pub commit: CommittedSubDag,
+    pub reputation_scores: Vec<(AuthorityIndex, u64)>,
```

Comment:
Considering changing this to an option and only sending scores on the commit after leader schedule change occurred. Right now the scores are submitted and swapped every commit which is redundant on the Sui side

Reply:
Yeah if from our tests we don't see any immediate performance degradation - which I doubt as the code that updates the scores doesn't do that much - then it would be preferably having a way to update the scores once on SUI side.

Reply:
What is the reason to not put `reputation_scores` into `CommittedSubDag`? I don't see other uses of `CommittedSubDag` besides sending to Sui.

Reply:
Good point, for some reason I thought it was user more than just for submission to Sui. Will add scores to `ComittedSubdag`

Reply:
Sounds good. `Commit` and `CommitInfo` are the only data structures used for storage and network.